test/stress1.c

```diff
@@ -22,13 +22,13 @@ int main(int argc, char **argv) {
 
 	srand (time (NULL));
 
-	for (i = 0; i < SDB_KSZ - 1; i++) {
+	for (i = 0; i < SDB_CDB_MAX_KEY - 1; i++) {
 		key[i] = 'A' + rand () % 26;
 		value[i] = 'a' + rand () % 26;
 	}
 
-	key[SDB_KSZ - 2] = '\0';
-	value[SDB_KSZ - 2] = '\0';
+	key[SDB_CDB_MAX_KEY - 2] = '\0';
+	value[SDB_CDB_MAX_KEY - 2] = '\0';
```

Comment:
why -2 ?
but ok for me.

Reply:
I don't even know if this test is run anywhere, will be obsolete soon anyway.